test/e2e/specs/all-features-confirmation/errors/errors.spec.js

```diff
@@ -337,107 +289,107 @@ describe('Error related test cases,', function() {
-              errModalClass =  chaisePage.recordPage.getModalText();
-              chaisePage.waitForElement(errModalClass);
-              return errModalClass.getText();
+              return chaisePage.recordPage.getConfirmDeleteButton().click().then(function() {
+                errModalClass =  chaisePage.recordPage.getModalText();
+                return chaisePage.waitForElement(errModalClass).then(function() {
+                  return errModalClass.getText();
+                });
+              });
           }).then(function (errorText) {
               // Added OR case to avoid discrepancy in error message when table is deleted
               expect(errorText == testParams.deletionErrTextBooking || errorText == testParams.deletionErrTextAccommodationImg).toBe(true, "409 Conflict could not be matched! Check conflict during deletion.");
+              done();
           }).catch(function(error) {
               console.log(error);
+              done.fail(err);
               expect('Something went wrong with this promise chain.').toBe('Please see error message.');
           });
       });
 
     });
 
-    describe("Error check for invalid paging changes in RecordSet", function(){
 
-      beforeAll(function() {
-          pageTestUrl = browser.params.url + "/recordset/#" + browser.params.catalogId + "/" + testParams.schemaName + ":" + testParams.table_name +  "/id=2002@after()";
-          browser.get(pageTestUrl);
-      });
+    describe("History for errorneous Url", function(){
 
-      it("should be returned Invalid Page Criteria", function (done) {
-          var modalTitle = chaisePage.recordEditPage.getModalTitle(),
-              modalActionBody = chaisePage.recordEditPage.getModalActionBody();
+        beforeAll(function() {
+            browser.ignoreSynchronization = true;
+            var url = browser.params.url + "/record/#" + browser.params.catalogId + "/" + testParams.schemaName + ":" + testParams.table_name +  "/id=269111";
+            browser.get(url);
+            chaisePage.waitForElement(element(by.css('.modal-dialog ')));
+        });
 
-          chaisePage.waitForElement(modalTitle).then(function(){
-              return modalTitle.getText();
-          }).then(function (text) {
-              expect(text).toBe(testParams.facetErrorstext.invalidPageCriteriaTitle, "Invalid Page Criteria error pop-up could not be opened!");
-              return modalActionBody.getText();
-          }).then(function (errorText) {
-              expect(errorText).toBe(testParams.facetErrorstext.invalidPageCriteriaBody, "Error action text did not match");
-              done();
-          }).catch(function(error) {
-              console.log(error);
-              done.fail();
-          });
-      });
+        it('After clicking back button initial page should appear', function(done){
+            chaisePage.recordPage.getErrorModalOkButton().then(function(btn){
```

Comment:
Before clicking on the ok button, we should add verification of the error title and message being present in the modal.

Reply:
It seems like we're testing this type of error in the other `describe`s. This is supposed to just test the "back" button in that case. I feel like we could just move this `it` to that `describe` that is testing that type of error.